please

```diff
@@ -61,9 +59,20 @@ if [ "$USE_NIX" = "1" ]; then
 
 else
   if [ "$USE_DOCKER" = "1" ]; then
+    DOCKER_HASH="`pwd``git rev-parse --abbrev-ref HEAD`"
+    DOCKER_NAME="mozilla-releng-services-v`cat ./VERSION`-`echo -n $DOCKER_HASH | sha256sum | cut -d ' ' -f 1`"
     if [ ! "$(docker ps -q -f name=$DOCKER_NAME)" ]; then
       if [ ! "$(docker ps -qa -f name=$DOCKER_NAME)" ]; then
-        docker run --privileged -td --name $DOCKER_NAME --volume="$(pwd)":/app --workdir=/app -p 7000:7000 -p 8000-8100:8000-8100 mozillareleng/services:base-`cat ./VERSION`
+        docker run \
+            --privileged \
+            -td \
+            --name $DOCKER_NAME \
+            --volume="$(pwd)":/app \
+            --volume=/var/run/docker.sock:/var/run/docker.sock \
```

Comment:
If i understand correctly, that's the main difference: you expose the docker socket inside the running container.
Do you need Write access to the socket ? Otherwise you can just append `:ro` to get a read-only socket.

Reply:
whatever the user has is ok.